personal_mycroft_backend/backend/precise.py

```diff
@@ -15,10 +15,10 @@ def precise_upload():
             fn = uploads[precisefile].filename
             if fn == 'audio':
                 name = (str(int(time.time()))) + ".wav"
-                uploads[precisefile].save(os.path.join(app.config['UPLOAD_FOLDER'], name))
+                uploads[precisefile].save(os.path.join(app.config['PRECISE_DATA_FOLDER'], name))
```

Comment:
```PRECISE_DATA_FOLDER``` is a global variable, it's not in ```app.config```, but maybe we should refactor the code to use ```app.config``` instead of the way current settings.py is?

Reply:
I'd have to sit down and review but might be a few days.

Reply:
that should be another PR and change the way settings.py are handled through the whole package, for now just pass ```PRECISE_DATA_FOLDER``` instead of ```app.config['PRECISE_DATA_FOLDER ']```